README.md

```diff
@@ -73,4 +73,12 @@ Then to run the container, run this command:
 docker run --env-file ./.env -p 5100:8000 -it --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app-v0.3:prod
 
 Note: Port 5100 can be changed to another local port if preferred, if you wish to run the container in the background please enter docker run -d at the beginning of the command
-Any updates made to the 'Todo_app' code will be automatically displayed within the browser once the webpage has been refreshed
\ No newline at end of file
+Any updates made to the 'Todo_app' code will be automatically displayed within the browser once the webpage has been refreshed
+
+If you wish to view context, container and component diagrams please view all files in the documentation folder of this repository. The .drawio files can be opened in Visual Studio Code and viewed providing you have installed the Draw.io Integration extension. 
+
+Basic tests have been added in tests and tests_end2end folder. Docker-compose.yml has been added so you can run docker compose up to launch a test docker container that will run the tests. You can launch the application by opening browser and going to http://localhost:5100/.
```

Comment:
I'd suggest adding a little formatting just to make the command stand out: using backticks (`` ` ``) to call out code for example can help to highlight code snippets `like this`


```suggestion
Basic tests have been added in tests and tests_end2end folder. Docker-compose.yml has been added so you can run `docker compose up` to launch a test docker container that will run the tests. You can launch the application by opening browser and going to http://localhost:5100/.
```